cloud/passportauth/integration/UserApi.ts

```diff
@@ -0,0 +1,40 @@
+import {BaseUser} from '../src/index';
+import {DataRepository} from './UserDataRepository';
+import {User} from './UserSeedData';
+
+/**
+ * A sample implementation of a base user api
+ */
+export class UserApi implements BaseUser {
```

Comment:
As we agreed before. This entire class is not really needed as it serve no purpose. 
We can:
Have user interface with set of requirements as methods:
```
getUserById
```

Have single method in repository to get user object.